docs/CodeOwners.md

```diff
@@ -0,0 +1,112 @@
+# Code Owners
+
+This file outlines the owners of the FHIR Server features in OSS.
+
+### Portal
+
+rojo (RB Johnson)
+
+patalada (Paul Taladay)
+
+### Custom Search & reindexing
+
+jaerwin (Jared Erwin)
+
+rotodd (Robin Todd)
+
+### Patient $everything
+
+rotodd (Robin Todd)
+
+ivmatan (Ivan Matantsev)
+
+### Sort
+
+namadabu (Narasimhan Madabusi)
```

Comment:
Probably shouldn't be on the list anymore.

Reply:
😢 removed in 4eac110.